tests/test_onnx/utils.py

```diff
@@ -32,14 +33,18 @@ def ort_validate(model, feats, onnx_io='tmp.onnx'):
     generated by torch.
 
     Args:
-        model (nn.Module): the model to be verified
-        feats (list(torch.Tensor) | torch.Tensor): the input of model
-        onnx_io (str): the name of onnx output file
+        model (nn.Module | function) : the function of model or model
+        to be verified.
+        feats (list(torch.Tensor) | torch.Tensor): the input of model.
+        onnx_io (str): the name of onnx output file.
     """
-    model.cpu().eval()
+    # if mdoel is a function, wrapped the function.
+    if (hasattr(model, '__call__')):
```

Comment:
Remove outside `()`

Reply:
```
else:
       wrap_model = model
```

Reply:
Verify if `hasattr(model, '__call__')` always return `True ` for normal function, instance of an `torch.nn.Module` or the `forward` method of an instance of an `torch.nn.Module`.